qutebrowser/browser/commands.py

```diff
@@ -1647,6 +1650,11 @@ def on_editing_finished(self, elem, text):
         except webelem.Error as e:
             raise cmdexc.CommandError(str(e))
 
+    def on_editor_orphaned(self, ed):
+        ed.editing_finished.disconnect()
+        ed.editing_finished.connect(
+            lambda: message.warning('Edited element was closed'))
```

Comment:
I'd prefer "Edited element vanished" or so (we already use that terminology elsewhere).